sdks/python/apache_beam/typehints/opcodes.py

```diff
@@ -175,33 +176,16 @@ def binary_subscr(state, unused_arg):
 binary_xor = inplace_xor = symmetric_binary_op
 binary_or = inpalce_or = symmetric_binary_op
 
-# As far as types are concerned.
-slice_0 = nop
-slice_1 = slice_2 = pop_top
-slice_3 = pop_two
-store_slice_0 = store_slice_1 = store_slice_2 = store_slice_3 = nop
-delete_slice_0 = delete_slice_1 = delete_slice_2 = delete_slice_3 = nop
-
 
 def store_subscr(unused_state, unused_args):
   # TODO(robertwb): Update element/value type of iterable/dict.
   pass
 
 
-binary_divide = binary_floor_divide = binary_modulo = symmetric_binary_op
-binary_divide = binary_floor_divide = binary_modulo = symmetric_binary_op
-binary_divide = binary_floor_divide = binary_modulo = symmetric_binary_op
-
-# print_expr
```

Comment:
I think these were in comments so as to have the entire set of opcodes represented here. (Maybe this could be worded more explicitly.)

Reply:
These comments have rotted over time, so going over the whole list of opcodes (https://docs.python.org/3/library/dis.html) would make sense at this point.